src/net.h

```diff
@@ -1025,22 +1006,16 @@ class CConnman
 
     /**
      * Do the read/write for connected sockets that are ready for IO.
-     * @param[in] nodes Nodes to process. The socket of each node is checked against
-     * `recv_set`, `send_set` and `error_set`.
-     * @param[in] recv_set Sockets that are ready for read.
-     * @param[in] send_set Sockets that are ready for send.
-     * @param[in] error_set Sockets that have an exceptional condition (error).
+     * @param[in] nodes Nodes to process. The socket of each node is checked against `what`.
+     * @param[in] what Sockets that are ready for IO.
      */
-    void SocketHandlerConnected(const std::vector<CNode*>& nodes,
-                                const std::set<SOCKET>& recv_set,
-                                const std::set<SOCKET>& send_set,
-                                const std::set<SOCKET>& error_set);
+    void SocketHandlerConnected(const std::vector<CNode*>& nodes, Sock::WaitData& what);
```

Comment:
af45eee `what` is an in-param
```suggestion
    void SocketHandlerConnected(const std::vector<CNode*>& nodes, const Sock::WaitData& what);
```

Reply:
Done.